bids/grabbids/config/bids.json

```diff
@@ -5,38 +5,39 @@
         {
             "name": "subject",
             "pattern": "[/\\\\]sub-([a-zA-Z0-9]+)",
-            "directory": "{{root}}/{subject}"
+            "directory": "{{root}}/{subject}",
+            "aliases": ["sub"]
         },
         {
             "name": "session",
             "pattern": "[_/\\\\]ses-([a-zA-Z0-9]+)",
             "mandatory": false,
             "directory": "{{root}}/{subject}/{session}",
-            "missing_value": "ses-1"
+            "missing_value": "ses-1",
+            "aliases": ["ses"]
         },
         {
             "name": "task",
             "pattern": "[_/\\\\]task-([a-zA-Z0-9]+)"
         },
         {
             "name": "acquisition",
-            "pattern": "[_/\\\\]acq-([a-zA-Z0-9]+)"
-        },
-        {
-            "name": "acq",
-            "pattern": "[_/\\\\]acq-([a-zA-Z0-9]+)"
+            "pattern": "[_/\\\\]acq-([a-zA-Z0-9]+)",
+            "aliases": ["acq"]
```

Comment:
I have a suspicion that this is causing the test failure:

```
layout2 = BIDS Layout: ...ackages/bids/tests/data/7t_trt | Subjects: 10 | Sessions: 20 | Runs: 20
    def test_filter_simple_variable(layout2):
        dataset = load_variables(layout2, types=['scans'])
        sessions = dataset.get_nodes('session')
        variables = [s.variables['surroundings'] for s in sessions]
        merged = merge_variables(variables)
>       assert merged.to_df().shape == (60, 9)
E       assert (60, 8) == (60, 9)
E         At index 1 diff: 8 != 9
E         Use -v to get the full diff
```

My guess is that we were getting a column for `acq` and `acquisition`, and now we're getting just `acquisition`. @tyarkoni is that likely? (I can dig into the actual test, if you don't know off the top of your head.)

Reply:
@tyarkoni Same failure.